### ISSUE TITLE
zpush directory does not exist (jessie branch)

### ISSUE THREAD

**Issue by ariddell** [sovereign/sovereign#469]
Fresh install hits a failure
```
TASK: [mailserver | Configure z-push apache alias and php settings] *********** 
msg: Destination directory /etc/apache2/conf.d does not exist
```

**Comment by ariddell** [comment]
It's ``conf-available`` now.

**Comment by mikeashley** [comment]
PR #472 fixes this.

**Comment by neuhaus** [comment]
merged #472
